README.md

```diff
@@ -2,22 +2,36 @@
 
 # Terraform Plugin Framework
 
-terraform-plugin-framework is an experimental module with no backwards compatibility guarantees, as indicated by its version 0 SemVer releases. We are mindful of our consumers and will strive to either maintain backwards compatibility or offer low-effort upgrade paths to provider developers, but our compatibility guarantees on this module are weaker than the [terraform-plugin-sdk](https://github.com/hashicorp/terraform-plugin-sdk) module. The purpose of this repository is to get real world experience with the design and implementation of this code before it is bound by the SDK's compatibility guarantees. This module is intentionally limited in its lifespan, and will become the next major version of the terraform-plugin-sdk module once were confident in its design and implementation, and this repository will be archived. When this happens, we will provide an upgrade path from users of the terraform-plugin-framework repository to the terraform-plugin-sdk release.
+terraform-plugin-framework is a module for building Terraform providers built on [terraform-plugin-go](https://github.com/hashicorp/terraform-plugin-go). It aims to provide as much of the power, predictability, and versatility of terraform-plugin-go as it can while abstracting away implementation details and repetitive, verbose tasks.
+
+## Status
+
+terraform-plugin-framework is still experimental. We are committed to moving forward with the module, but cannot guarantee any of its interfaces will not change as long as it is in version 0. We're waiting for more feedback, usage, and maturity before we're comfortable committing to APIs with the same years-long support timelines that [terraform-plugin-sdk](https://github.com/hashicorp/terraform-plugin-sdk) brings.
```

Comment:
we created a fancy badge for experimental things for k8s- alpha - https://camo.githubusercontent.com/8ad47215ae8b556345c074d2636cdf5e8a7f54068c110d1a1795501b43fab52e/68747470733a2f2f696d672e736869656c64732e696f2f62616467652f7374617475732d6578706572696d656e74616c2d454141413332